modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/AbstractGeoIpIT.java

```diff
@@ -61,4 +64,22 @@ protected Settings nodeSettings(final int nodeOrdinal) {
             return Collections.singletonList(Setting.simpleString("ingest.geoip.database_path", Setting.Property.NodeScope));
         }
     }
+
+    @After
+    public void cleanDatabases() throws Exception {
+        super.tearDown();
+        Path path = internalCluster().getInstance(Environment.class).tmpFile();
```

Comment:
In case that there are multiple nodes, we need to use this method to get all `Environment` instances and get each tmpFile: `internalCluster().getInstances(Environment.class)`

Reply:
after discussing offline we will use different approach here